editor/actions.js

```diff
@@ -41,3 +57,42 @@ export function savePost( dispatch, postId, edits ) {
 		} );
 	} );
 }
+
+export function mergeBlocks( dispatch, blockA, blockB ) {
```

Comment:
Yes not as easy as `concatChildren`, because merging blocks could produce several blocks, merging only the first paragraph. I'm thinking we could do this recursively, but I don't see much value to support this for now.